kioku.cabal

```diff
@@ -41,7 +41,7 @@ library
                      , filepath
                      , mmap
                      , vector
-                     , vector-algorithms < 0.8.0.0
+                     , vector-algorithms
```

Comment:
This probably needs a lower bound now, right?

Reply:
Done